blob/rollup.test.config.js

```diff
@@ -0,0 +1,16 @@
+import multi from 'rollup-plugin-multi-entry';
+import baseConfig from './rollup.config';
+const [node, browser] = baseConfig;
+
+node.input = ['dist-esm/test/*.js', 'dist-esm/test/node/*.js'];
```

Comment:
Why do we need build test for node?

`mocha --no-timeouts dist-esm/test/*.js dist-esm/test/node/*.js` doesn't use this, right?